### ISSUE TITLE
Missing types: UserV2.public_metrics.like_count, TweetV2.public_metrics.bookmark_count

### ISSUE THREAD

**Issue by liorgrossman** [PLhery/node-twitter-api-v2#527]
I started using node-twitter-api-v2 recently and I found the bundled types for responses _extremely helpful_.

However, I did encounter a couple of fields that are available through the Twitter API, but do not exist in the types:

1. In `UserV2`, under `public_metrics`, the field `like_count` should probably be added.
Here's the real data returned from the Twitter V2 API when this field is requested:
```js
  "public_metrics": {
            "followers_count": 161,
            "following_count": 168,
            "tweet_count": 159,
            "listed_count": 0,
            "like_count": 437
          },
```
2. In `TweetV2`, under `public_metrics` (which is of type `TweetPublicMetricsV2`), the field `bookmark_count` should probably be added.
Here's real data returned from the Twitter V2 API when this field is requested:
```js
  "public_metrics": {
            "retweet_count": 116,
            "reply_count": 0,
            "like_count": 513,
            "quote_count": 3,
            "bookmark_count": 31,
            "impression_count": 7668
          },
```

I'm able to get the code to work and return these fields, but TypeScript obviously does not like it:
<img width="933" alt="image" src="https://github.com/PLhery/node-twitter-api-v2/assets/987284/822673eb-ea16-41b4-9c3d-c31f046d54b4">

Please let me know if you have any questions.

**Comment by solojungle** [comment]
@liorgrossman created a PR for this issue!

**Comment by PLhery** [comment]
Published in 1.16.4
